Extending/Property-Editors/Property-Actions/index.md

```diff
@@ -47,8 +90,7 @@ angular.module('umbraco').component('myPropertyEditor', {
 });
 ```
 
-## Implementing a Property Action
-The implementation of Property Actions consists of two parts. First a definition of the actions and secondly parsing the lists of actions to `umbProperty.setPropertyActions`.
+See the following example for implementation of Property Actions in a Controller, parsing actions to `this.umbProperty.setPropertyActions`.
```

Comment:
I'm a little confused about this one.
Shouldn't this be part of the `### Controller Implementation` section?

Reply:
Oh, was a mistake, should say Component...